src/python_testing/matter_testing_support.py

```diff
@@ -265,6 +265,15 @@ def wait_for_event_report(self, expected_event: ClusterObjects.ClusterEvent, tim
         asserts.assert_equal(res.Header.EventId, expected_event.event_id, "Expected event ID not found in event report")
         return res.Data
 
+    def wait_for_no_event_report(self, timeout: int = 10):
+        """This function allows a test script to block waiting for the specific event to arrive with a timeout.
+           It returns the event data so that the values can be checked."""
+        try:
+            res = self._q.get(block=True, timeout=timeout)
+        except queue.Empty:
+            return
+
+        asserts.assert_equal(False, f"Event reported when not expected {res}")
```

Comment:
This will raise an exception because you are missing the 2nd arg. i.e. assert that False == True But you're asserting False == f"Event..."

maybe
```

        try:
            res = self._q.get(block=True, timeout=timeout)
        except queue.Empty:
            return

        asserts.fail(f"Event reported when not expected {res}")


```